src/EditorFeatures/Core/Extensibility/Composition/VisualStudioVersionMetadata.cs

```diff
@@ -1,16 +0,0 @@
-﻿using System;
-using System.Collections.Generic;
-using Roslyn.Utilities;
-
-namespace Microsoft.CodeAnalysis.Editor.Extensibility.Composition
-{
-    internal sealed class VisualStudioVersionMetadata
```

Comment:
@CyrusNajmabadi We can bring it back if needed. And TBH, after doing this exercise, I'm not sure I'd say we should try this approach again. With the VS Installer now allowing side-by-side with much less drama, just having separate branches might be a lot easier.